bundles/persistence/org.eclipse.smarthome.persistence.mapdb/src/main/java/org/eclipse/smarthome/persistence/mapdb/internal/MapDbItem.java

```diff
@@ -32,29 +35,29 @@
     private Date timestamp;
 
     @Override
-    public String getName() {
+    public @NonNull String getName() {
```

Comment:
Instead of `@NonNull` please use `@NonNullByDefault` on class level and then `@Nullable` on the exceptions from the default. Read this part of the documentation for reasons: https://www.eclipse.org/smarthome/documentation/development/guidelines.html#a-code-style